deepchem/models/torch_models/chemberta.py

```diff
@@ -0,0 +1,174 @@
+"""
+Huggingface/transformers RoBERTa model for sequence-based property prediction.
+"""
+
+from numpy.lib.npyio import save
+import torch.nn as nn
```

Comment:
These should be wrapped in a try/catch block (since pytorch is an optional dependency)

Reply:
Will do - nice catch!